tests/tests-api.php

```diff
@@ -187,6 +189,9 @@ public function setUp() {
 	public function tearDown() {
 		parent::tearDown();
 
+		// Revoke key to ensure `update_key()` will generate a new one.
+		EDD()->api->revoke_api_key( self::$user_id );
```

Comment:
Added this due to numerous calls in tests to `EDD()->api->update_key( self::$user_id );` . The issue was that if the API key already existed when `EDD()->api->update_key( self::$user_id );` was run, the key was actually just being revoked and not generated.